ts/screens/authentication/LandingScreen.tsx

```diff
@@ -9,11 +9,11 @@ import { NavigationScreenProp, NavigationState } from "react-navigation";
 import { connect } from "react-redux";
 
 import { HorizontalScroll } from "../../components/HorizontalScroll";
-import { LandingCardComponent } from "../../components/LandingCard";
+import { LandingCardComponent } from "../../components/LandingCardComponent";
 import BaseScreenComponent from "../../components/screens/BaseScreenComponent";
 import IconFont from "../../components/ui/IconFont";
 
-import { isDevEnvironment } from "../../config";
+import { isCIEauthenticationEnabled, isDevEnvironment } from "../../config";
```

Comment:
remove empty spaces among imports to get them being ordered :)

Reply:
Done!